Signal/src/view controllers/SignalsViewController.m

```diff
@@ -339,25 +339,32 @@ - (void)tableViewCellTappedDelete:(NSIndexPath *)indexPath {
                       preferredStyle:UIAlertControllerStyleAlert];
         [self presentViewController:removingFromGroup animated:YES completion:nil];
```

Comment:
I thought that giving some feedback to the user was a good thing.

Can you confirm you don't want the alert, I will push the updated code.